File: src/projectecommerce/Product.java
package projectecommerce;

public class Product {
	private final String id;
	private String title;
	private int price;
	
	public Product(String id, String title, int price) {
		this.id = id;
		this.title = title;
		this.price = price;
	}

	public String getId() {
		return id;
	}

	public String getTitle() {
		return title;
	}

	public int getPrice() {
		return price;
	}

	@Override
	public String toString() {
		return "Product [id=" + id + ", title=" + title + ", price=" + price + "]";
	}
	
	
	
}

File: src/typescript-ecommerce/src/product.ts

export class Product {
    private id: string;
    private title: string;
    private price: number;

    constructor(id: string, title: string, price: number) {
        this.id = id;
        this.title = title;
        this.price = price;
    }

    getId(): string {
        return this.id;
    }

    getTitle(): string {
        return this.title;
    }

    getPrice(): number {
        return this.price;
    }

    toString(): string {
        return `Product [id=${this.id}, title=${this.title}, price=${this.price}]`;
    }

}
